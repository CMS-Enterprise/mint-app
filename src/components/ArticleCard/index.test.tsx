// import React from 'react';
// import { MemoryRouter } from 'react-router-dom';
// import { render } from '@testing-library/react';

// import itGovernanceArticles from 'views/Help/ITGovernance/articles';
// import section508Articles from 'views/Help/Section508/articles';

// import ArticleCard from './index';

// describe('RelatedArticle', () => {
//   it('matches the snapshot', () => {
//     const { asFragment } = render(
//       <MemoryRouter>
//         <ArticleCard {...itGovernanceArticles[0]} />
//       </MemoryRouter>
//     );
//     expect(asFragment()).toMatchSnapshot();
//   });

//   it('renders correct itgovernance article type', () => {
//     const { getByText } = render(
//       <MemoryRouter>
//         <ArticleCard {...itGovernanceArticles[0]} />
//       </MemoryRouter>
//     );

//     expect(getByText('IT Governance')).toBeInTheDocument();
//   });

//   it('renders correct 508 article type', () => {
//     const { getByText } = render(
//       <MemoryRouter>
//         <ArticleCard {...section508Articles[0]} />
//       </MemoryRouter>
//     );

//     expect(getByText('Section 508')).toBeInTheDocument();
//   });

//   it('renders Article Card entirely wrapped as a link', () => {
//     const { container } = render(
//       <MemoryRouter>
//         <ArticleCard {...section508Articles[0]} isLink />
//       </MemoryRouter>
//     );

//     expect(
//       container.getElementsByClassName('article-card--isLink').length
//     ).toBe(1);
//   });
// });
