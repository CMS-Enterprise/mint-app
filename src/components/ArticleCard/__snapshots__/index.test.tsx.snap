// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RelatedArticle matches the snapshot 1`] = `
<DocumentFragment>
  <li
    class="usa-card desktop:grid-col-4 article"
    data-testid="article-card"
  >
    <div
      class="usa-card__container radius-md shadow-2 minh-mobile"
    >
      <header
        class="usa-card__header padding-x-3 padding-top-3"
        data-testid="CardHeader"
      >
        <h3
          class="line-height-body-4 margin-bottom-1"
        >
          Prepare for the Governance Review Team meeting
        </h3>
        <a
          class="usa-link display-inline-block margin-bottom-1"
          href="/help/it-governance"
        >
          <span
            class="easi-tag system-profile__tag text-primary-dark bg-primary-lighter padding-bottom-1"
            data-testid="tag"
          >
            IT Governance
          </span>
        </a>
      </header>
      <div
        class="usa-card__body padding-top-0 article__body"
        data-testid="CardBody"
      >
        <p>
          Learn what you need in order to be successful, including what to expect during and after the meeting, what to prepare for, and what to bring to the meeting.
        </p>
      </div>
      <div
        class="usa-card__footer padding-top-2 article__footer"
        data-testid="CardFooter"
      >
        <a
          class="usa-link easi-request__button-link padding-x-2"
          href="/"
        >
          Read
        </a>
      </div>
    </div>
  </li>
</DocumentFragment>
`;
